api/migrations/0001_initial.py

```diff
@@ -196,6 +196,23 @@ class Migration(migrations.Migration):
                 'abstract': False,
             },
         ),
+        migrations.CreateModel(
+            name='Year',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('createdAt', models.DateTimeField()),
+                ('updatedAt', models.DateTimeField()),
+                ('theme', models.TextField(blank=True, verbose_name='Theme of this year')),
```

Comment:
Please use the name "topic". Theme can be misleading, because it is also connected with visual styling.